mobile/RedOctober/RedOctober/OptionViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  OptionViewController.swift
+//  RedOctober
+//
+//  Created by Longfei Li on 15/12/5.
+//  Copyright © 2015年 Cody. All rights reserved.
+//
+
+import UIKit
+
+class OptionViewController: UIViewController {
+
+    @IBOutlet weak var addrText: UITextField!
+    @IBOutlet weak var enterButton: UIButton!
+    @IBOutlet weak var portText: UITextField!
+    @IBOutlet weak var serverAddr: UILabel!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.serverAddr.text! = Server.endpoint
+        self.enterButton.layer.cornerRadius = 5
+        self.enterButton.layer.borderWidth = 1
+        self.enterButton.layer.borderColor = UIColor.whiteColor().CGColor
+        self.portText.placeholder = "ex.8080"
+        // Do any additional setup after loading the view.
+    }
+    
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
```

Comment:
It is not necessary, this override function was just there when this ViewController class was created, but it can be removed.